Assets/Scripts/PlayerSelect/PlayerEyeManager.cs

```diff
@@ -13,14 +14,21 @@ public void Start()
     {
         // Find eye material
         SkinnedMeshRenderer eyeRenderer = transform.GetChild(0).GetComponent<SkinnedMeshRenderer>();
-        eyeMaterial = eyeRenderer.materials[1];
-        resetEyePosition();
+        Material[] playerMaterials = eyeRenderer.materials;
+        foreach (Material playerMaterial in playerMaterials)
+        {
+            if (playerMaterial.name == "PlayerEyesMaterial (Instance)") { 
```

Comment:
Could be accomplished with a .Single(m => m.name = "blah blah")

Reply:
Fixed, and re-added cards on table. I had completely overlooked them when reimporting the new model. Thank you!